code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -322,16 +322,8 @@
 	for(var/explodee in GLOB.player_list)
 		SEND_SOUND(explodee, doomsday_alarm)
 	sleep(100)
-	for(var/mob/living/L in GLOB.mob_list)
-		var/turf/T = get_turf(L)
-		if(!T || T.z != z_level)
-			continue
-		if(issilicon(L))
-			continue
-		to_chat(L, "<span class='danger'><B>The blast wave from [src] tears you atom from atom!</B></span>")
-		L.dust()
+	SSticker.station_explosion_cinematic(null, "AI malfunction")
```

Comment:
Moving the cinematic *before* the `sleep` might produce better results as the video and audio syncs up, worth a try?

Reply:
I personally think it works better when the sleep is after the sound plays. It'll give the "0.8 SECONDS UNTIL DOOMSDAY DEVICE ACTIVATION!" announcement, alarms will start playing and and then it cuts to an external shot of the station exploding. The nuke handles the sound/cinematic in the same way as well.